jupiterone/questions/questions.yaml

```
@@ -1,4 +1,15 @@
 ---
 sourceId: managed:aquasec
 integrationDefinitionId: "${integration_definition_id}"
-questions: []
+questions: 
+- id: integration-question-Acquasec-user-that-send-security-results-or-sendNewRisks
+  title: Which Acquasec user that sendScanResults or sendNewRisks are set to true?
+  description:
+    Returns all Acquasec users that sendScanResults or sendNewRisks.
```

```suggestion
  title: Which Aqua Security user that sendScanResults or sendNewRisks are set to true?
  description:
    Returns all Aqua Security users that sendScanResults or sendNewRisks.
```
Needs one small change to the vendor name.

---

Thanks for the clarity, it's appreciated for future reference.